api/resources_portal/models/user.py

```diff
@@ -16,6 +16,10 @@ class User(AbstractUser):
 
     organizations = models.ManyToManyField("Organization", through="OrganizationUserAssociation")
 
+    def save(self, *args, **kwargs):
+        self.id = uuid.UUID(str(self.id))
+        super(User, self).save(*args, **kwargs)
```

Comment:
Did this end up being necessary? I think I was pairing with you while you wrote this and we did it to address a red herring IRRC. Can you try taking it out and seeing if everything still works?

Reply:
Just tested this- it still seems to be neccesary.